packages/user-service/src/resolver.ts

```diff
@@ -5,94 +5,98 @@ import { UserGroupAPIHelper } from './helpers';
 
 export const UserGroupResolver = {
   Query: {
-    getUser(root: any, args: any, ctx: any) {
-      return User.findOne({ uid: args.uid })
-        .then(data => data)
-        .catch(err => err);
+    getUser ( root: any, args: any, ctx: any ) {
```

Comment:
Also, once we have the new query for `getUsersBy`, we can deprecate this as the new query can do the same job.

Reply:
Updated